services/static-webserver/client/source/class/osparc/desktop/MainPage.js

```diff
@@ -85,9 +85,17 @@ qx.Class.define("osparc.desktop.MainPage", {
       if (this.__studyEditor) {
         const preferencesSettings = osparc.desktop.preferences.Preferences.getInstance();
         if (preferencesSettings.getConfirmBackToDashboard()) {
-          const msg = this.tr("Do you really want to save and close the study?");
+          let msg = this.tr("Do you really want to save and close the study?");
+          let confirmText = this.tr("Save & Close");
+          if (osparc.utils.Utils.isProduct("s4llite")) {
+            msg = this.tr("Do you really want to close the project?");
+            msg += "<br>";
+            msg += this.tr("Make sure you saved the changes to the current smash file and open notebooks");
```

Comment:
Suggestion: idea below is to highlight the things the user should check (not sure if bold is enough).
```suggestion
            msg += this.tr("Make sure you saved the changes to the current <b>smash file</b> and <b>open notebooks</b>");
```